charts/latest/csi-driver-nfs/templates/snapshotclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.externalSnapshotter.create }}
```

Comment:
remove this since `VolumeSnapshotClass` does not need to be dependent on externalSnapshotter, use `volumeSnapshotClass.create` instead

Reply:
Alright